schemas/src/AdaptiveCard.json

```diff
@@ -0,0 +1,80 @@
+{
+	"$schema": "https://raw.githubusercontent.com/microsoft/AdaptiveCards/6f39aedce45864ae1067ed44a5551dc973790bb5/source/nodejs/typed-schema/schema/lib/Type.json",
+	"description": "An Adaptive Card",
+	"properties": {
+		"version": {
+			"type": "string",
+			"description": "Schema version that this card requires. If a client is **lower** than this version, the `fallbackText` will be rendered. NOTE: Version is not required for cards within an `Action.ShowCard`. However, it *is* required for the top-level card.",
+			"examples": [
+				"1.0",
+				"1.1",
+				"1.2"
+			]
+		},
+		"body": {
+			"type": "Element[]",
+			"description": "The card elements to show in the primary card region."
+		},
+		"actions": {
+			"type": "Action[]",
+			"description": "The Actions to show in the card's action bar."
+		},
+		"selectAction": {
+			"type": "ISelectAction",
+			"description": "An Action that will be invoked when the card is tapped or selected. `Action.ShowCard` is not supported.",
+			"version": "1.1"
+		},
+		"style": {
+			"type": "ContainerStyle",
+			"description": "Style hint for the Adaptive Card.",
+			"version": "1.2"
+		},
+		"fallbackText": {
+			"type": "string",
+			"description": "Text shown when the client doesn't support the version specified (may contain markdown)."
+		},
+		"backgroundImage": {
+			"type": "BackgroundImage",
+			"description": "Specifies the background image of the card.",
+			"version": "1.2",
+			"shorthands": [
+				{
+					"type": "uri-reference",
+					"description": "The URL (or data url) to use as the background image. Supports data URI in version 1.2+",
+					"version": "1.0"
+				}
+			]
+		},
+		"minHeight": {
+			"type": "string",
+			"description": "Specifies the minimum height of the card.",
+			"examples": [
+				"50px"
+			],
+			"version": "1.2",
+			"features": [ 2293 ]
```

Comment:
What does this reference?

Reply:
It's something I was experimenting with as part of spec'ing, that way we can easily tie back a property to the feature it enables. So it references the issue # of the feature.

Currently not used anywhere, other than if we wanted to see what properties a feature added, we could search for the feature number and see it light up in the schema files theoretically